### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by cexbrayat** [Ninja-Squad/ninja-squad.github.com#291]
Getting links from: https://blog.ninja-squad.com/
├─BROKEN─ https://blog.ninja-squad.com/2017/11/02/what-is-new-in-angular-5 (HTTP_404)
├─BROKEN─ https://blog.ninja-squad.com/2018/05/04/what-is-new-in-angular-6 (HTTP_404)
├─BROKEN─ https://blog.ninja-squad.com/2018/07/27/what-is-new-angular-6.1 (HTTP_404)

Getting links from: https://blog.ninja-squad.com/2018/10/18/what-is-new-angular-7
├─BROKEN─ https://blog.ninja-squad.com/2017/11/02/what-is-new-in-angular-5 (HTTP_404)
├─BROKEN─ https://blog.ninja-squad.com/2018/05/04/what-is-new-in-angular-6 (HTTP_404)
├─BROKEN─ https://blog.ninja-squad.com/2018/07/27/what-is-new-angular-6.1 (HTTP_404)
Finished! 69 links found. 66 excluded. 3 broken.

Getting links from: https://blog.ninja-squad.com/2018/10/18/what-is-new-angular-7/#disqus_thread
├─BROKEN─ https://blog.ninja-squad.com/2017/11/02/what-is-new-in-angular-5 (HTTP_404)
├─BROKEN─ https://blog.ninja-squad.com/2018/05/04/what-is-new-in-angular-6 (HTTP_404)
├─BROKEN─ https://blog.ninja-squad.com/2018/07/27/what-is-new-angular-6.1 (HTTP_404)
Finished! 69 links found. 66 excluded. 3 broken.

Getting links from: https://blog.ninja-squad.com/2017/09/14/angular-cli-1.4/
├─BROKEN─ https://github.com/angular/devkit/tree/master/packages/angular_devkit/schematics (HTTP_404)
├─BROKEN─ https://github.com/angular/devkit/tree/master/packages/schematics/angular (HTTP_404)
Finished! 61 links found. 52 excluded. 2 broken.

Getting links from: https://blog.ninja-squad.com/2017/08/10/angular-cli-1.3/
├─BROKEN─ https://github.com/angular/devkit/tree/master/packages/angular_devkit/build_optimizer (HTTP_404)
Finished! 62 links found. 53 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2016/01/20/become-a-ninja-angular-2-ebook-release/
└─BROKEN─ http://www.docteursouris.fr/ (BLC_UNKNOWN)
Finished! 58 links found. 51 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2016/01/19/deviens-un-ninja-angular-2-sortie-ebook/
├─BROKEN─ http://www.docteursouris.fr/ (HTTP_undefined)
Finished! 58 links found. 54 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2015/10/27/angular-connect/
├─BROKEN─ http://debug.elm-lang.org/ (ERRNO_ENOTFOUND)
Finished! 56 links found. 47 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2015/04/14/angular2-ebook/
├─BROKEN─ https://books.ninja-squd.com/angularjs (ERRNO_ENOTFOUND)
Finished! 60 links found. 51 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2015/01/23/ebook-angularjs-dons-docteur-souris/
├─BROKEN─ http://www.docteursouris.fr/ (HTTP_undefined)
Finished! 58 links found. 53 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2014/10/28/nouveautes-angularjs-1.3-part-4/
├─BROKEN─ http://hypedrivendev.wordpress.com/2014/10/28/nouveautes-angularjs-1.3-part-4/ (HTTP_404)
Finished! 59 links found. 52 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2014/09/23/bilan-des-ventes-ebook-angularjs/
├─BROKEN─ http://www.docteursouris.fr/ (HTTP_undefined)
Finished! 68 links found. 54 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2014/09/02/devenez-un-ninja-avec-angularjs-ebook/
├─BROKEN─ http://www.docteursouris.fr/ (HTTP_undefined)
Finished! 55 links found. 49 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2014/08/28/nouveautes-angularjs-1.3-part-3/
├─BROKEN─ http://hypedrivendev.wordpress.com/2014/08/28/nouveautes-angularjs-1.3-part-3/ (HTTP_404)
Finished! 55 links found. 49 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2014/06/24/nouveautes-angularjs-1.3/
└─BROKEN─ http://www.whatwg.org/specs/web-apps/current-work/multipage/the-input-element.html (HTTP_404)
Finished! 56 links found. 49 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2014/03/25/jhipster-boost-your-spring-angular-app/
├─BROKEN─ http://metrics.codahale.com/ (ERRNO_ENOTFOUND)
├─BROKEN─ http://yeoman.io/community-generators.html (HTTP_404)
Finished! 88 links found. 55 excluded. 2 broken.

Getting links from: https://blog.ninja-squad.com/2013/10/08/dates-de-formation/
├─BROKEN─ https://ninja-squad.fr/formations/formation-javaAdvanced (HTTP_404)
Finished! 55 links found. 51 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2015/03/17/quizzie-ops/
├─BROKEN─ https://quizzie.io/ (ERRNO_ECONNREFUSED)
Finished! 57 links found. 50 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2017/07/04/ninja-squad-a-5-ans/
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
Finished! 78 links found. 60 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2013/09/20/un-an-en-chiffres/
├─BROKEN─ http://realtimeconf.eu/ (HTTP_404)
Finished! 65 links found. 54 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2013/08/22/ninja-squad-1-an/
├─BROKEN─ https://ninja-squad.fr/formations/formation-java (HTTP_404)
├─BROKEN─ https://ninja-squad.fr/formations/formation-javaAdvanced (HTTP_404)
├─BROKEN─ http://golo-lang.org/news/2013/04/29/viral-marketing-thanks-to-ninjasquad/ (HTTP_404)
Finished! 58 links found. 47 excluded. 3 broken.

Getting links from: https://blog.ninja-squad.com/2013/11/29/bitbucket-vs-github/
├─BROKEN─ https://confluence.atlassian.com/display/BITBUCKET/Use+the+JIRA+DVCS+Connector+Plugin (HTTP_404)
├─BROKEN─ https://github.com/benbalter/dc-maps/blob/master/embassies.geojson (HTTP_404)
Finished! 70 links found. 48 excluded. 2 broken.

Getting links from: https://blog.ninja-squad.com/2013/03/22/nossii-devoxx-france/
├─BROKEN─ http://www.devoxx.com/display/FR13/BOF+Not+Only+SSII (HTTP_404)
├─BROKEN─ http://hopwork.com/ (HTTP_403)
Finished! 62 links found. 52 excluded. 2 broken.

Getting links from: https://blog.ninja-squad.com/2013/01/07/getting-real-book-review/
├─BROKEN─ http://hypedrivendev.wordpress.com/2012/12/27/getting-real-a-book-review/ (HTTP_404)
Finished! 54 links found. 48 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2013/02/08/code-story/
├─BROKEN─ http://code-story.net/ (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/about/david.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/about/jean-laurent.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/2013/02/01/concours-2013-phase-1.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/2013/02/02/jajascript.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/2013/01/22/scalaskel.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/2013/02/02/jajascript.html (HTTP_undefined)
Finished! 67 links found. 49 excluded. 7 broken.

Getting links from: https://blog.ninja-squad.com/2014/04/08/gagnez-votre-place-pour-mix-it/
├─BROKEN─ http://t.co/eTBqmzFLLP (ERRNO_ENOTFOUND)
Finished! 62 links found. 52 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2012/11/22/devoxx/
├─BROKEN─ http://weblogs.java.net/blog/chet/archive/2008/01/crystal_methodo.html (HTTP_503)
├─BROKEN─ http://dvcs.w3.org/hg/webcomponents/raw-file/tip/explainer/index.html (HTTP_404)
├─BROKEN─ https://plus.google.com/111962077049890418486 (HTTP_404)
├─BROKEN─ https://plus.google.com/114128610730314333831/posts/HEZBavvQqCF (HTTP_403)
Finished! 83 links found. 44 excluded. 4 broken.

Getting links from: https://blog.ninja-squad.com/2012/09/17/jugsummercamp-2012/
├─BROKEN─ http://a0.twimg.com/profile_images/952785929/summercamp7.png (ERRNO_ENOTFOUND)
├─BROKEN─ http://www.google.fr/tv/ (HTTP_404)
├─BROKEN─ http://twitter/audrey_neveu (ERRNO_ENOTFOUND)
├─BROKEN─ http://www.web-tambouille.fr/tag/Node.js (HTTP_404)
├─BROKEN─ http://pivotal.github.com/jasmine/ (HTTP_404)
Finished! 100 links found. 55 excluded. 5 broken.

Getting links from: https://blog.ninja-squad.com/2012/11/27/become-a-ninja/
├─BROKEN─ https://ninja-squad.com/formations/formation-java (HTTP_404)
├─BROKEN─ https://ninja-squad.com/formations/formation-javaAdvanced (HTTP_404)
Finished! 59 links found. 50 excluded. 2 broken.

Getting links from: https://blog.ninja-squad.com/2013/01/17/ninjackaton-lambda/
├─BROKEN─ https://plus.google.com/107576172998205518534/about (HTTP_404)
Finished! 51 links found. 44 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2012/10/19/java-championne/
├─BROKEN─ http://java.net/website/java-champions/bios.html#Crepet (HTTP_503)
Finished! 57 links found. 49 excluded. 1 broken.

Getting links from: https://blog.ninja-squad.com/2015/03/11/quizzie-spring/
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
├─BROKEN─ https://quizzie.io/badges/player (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/badges/champion (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/badges/pioneer (ERRNO_ECONNREFUSED)
Finished! 58 links found. 48 excluded. 4 broken.

Getting links from: https://blog.ninja-squad.com/2015/03/03/quizzie-announcement/
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
├─BROKEN─ https://quizzie.io/quizzes/50352/all-you-need-to-know-about-creating-a-quiz (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/tags (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/quizzes/50305/markdown---how-to-improve-the-look-of-your-quizzes (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/users (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/badges/categories/all (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
Finished! 58 links found. 46 excluded. 7 broken.

Getting links from: https://blog.ninja-squad.com/2015/03/03/annonce-quizzie/
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
├─BROKEN─ https://quizzie.io/quizzes/50350/tout-ce-quil-faut-savoir-sur-la-creation-de-quiz (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/tags (HTTP_undefined)
├─BROKEN─ https://quizzie.io/quizzes/50300/markdown-ou-comment-ameliorer-la-presentation-de-vos-quiz (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/users (HTTP_undefined)
├─BROKEN─ https://quizzie.io/badges/categories/all (HTTP_undefined)
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
Finished! 58 links found. 47 excluded. 7 broken.

Finished! 9877 links found. 8482 excluded. 75 broken.
Elapsed time: 7 minutes, 20 seconds

**Comment by clacote** [comment]
Internal links on the blog itself fixed in #292.

**Comment by cexbrayat** [comment]
What should we do about quizzie.io? docteursouris?

**Comment by clacote** [comment]
I guess this is not a problem to link a now dead website, when it simply disappears. Don’t you think so?

**Comment by cexbrayat** [comment]
On one side yeah I agree. On another, the domains might be used by someone else in the near future, with content we don't approve... Maybe we should just remove the links, and add a disclaimer in the articles about Quizzie that we discontinued the experiment.

**Comment by clacote** [comment]
Ah yes, makes sense for the reclaimed domains. Will do this for Quizzie ones, and simply remove the links to Docteur Souris.

**Comment by clacote** [comment]
After #293, here are the remaining broken links:
```
➜  blog git:(fix/broken-links) npx broken-link-checker --recursive --get --follow http://localhost:4000 | grep BROKEN
npx: installed 102 in 4.922s
└─BROKEN─ http://www.docteursouris.fr/ (BLC_UNKNOWN)
├─BROKEN─ http://www.docteursouris.fr/ (HTTP_undefined)
├─BROKEN─ http://www.docteursouris.fr/ (HTTP_undefined)
├─BROKEN─ http://www.docteursouris.fr/ (HTTP_undefined)
├─BROKEN─ http://www.docteursouris.fr/ (HTTP_undefined)
└─BROKEN─ https://quizzie.io/ (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
├─BROKEN─ http://realtimeconf.eu/ (HTTP_404)
├─BROKEN─ https://confluence.atlassian.com/display/BITBUCKET/Use+the+JIRA+DVCS+Connector+Plugin (HTTP_404)
├─BROKEN─ https://github.com/benbalter/dc-maps/blob/master/embassies.geojson (HTTP_404)
├─BROKEN─ http://www.devoxx.com/display/FR13/BOF+Not+Only+SSII (HTTP_404)
├─BROKEN─ http://hopwork.com/ (HTTP_403)
├─BROKEN─ http://code-story.net/ (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/about/david.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/about/jean-laurent.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/2013/02/01/concours-2013-phase-1.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/2013/02/02/jajascript.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/2013/01/22/scalaskel.html (ERRNO_ENOTFOUND)
├─BROKEN─ http://code-story.net/2013/02/02/jajascript.html (HTTP_undefined)
├─BROKEN─ http://t.co/eTBqmzFLLP (ERRNO_ENOTFOUND)
├─BROKEN─ http://weblogs.java.net/blog/chet/archive/2008/01/crystal_methodo.html (HTTP_503)
├─BROKEN─ http://dvcs.w3.org/hg/webcomponents/raw-file/tip/explainer/index.html (HTTP_404)
├─BROKEN─ https://plus.google.com/111962077049890418486 (HTTP_404)
├─BROKEN─ https://plus.google.com/114128610730314333831/posts/HEZBavvQqCF (HTTP_403)
└─BROKEN─ http://www.slideshare.net/svenpeters/jfokus-how-to-make-good-teams-great (BLC_UNKNOWN)
├─BROKEN─ http://a0.twimg.com/profile_images/952785929/summercamp7.png (ERRNO_ENOTFOUND)
├─BROKEN─ http://twitter/audrey_neveu (ERRNO_ENOTFOUND)
├─BROKEN─ http://www.google.fr/tv/ (HTTP_404)
├─BROKEN─ http://www.web-tambouille.fr/tag/Node.js (HTTP_404)
├─BROKEN─ http://pivotal.github.com/jasmine/ (HTTP_404)
├─BROKEN─ https://plus.google.com/107576172998205518534/about (HTTP_404)
└─BROKEN─ http://java.net/website/java-champions/bios.html#Crepet (HTTP_503)
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
├─BROKEN─ https://quizzie.io/badges/player (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/badges/champion (ERRNO_ECONNREFUSED)
└─BROKEN─ https://quizzie.io/badges/pioneer (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
├─BROKEN─ https://quizzie.io/quizzes/50352/all-you-need-to-know-about-creating-a-quiz (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/tags (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/quizzes/50305/markdown---how-to-improve-the-look-of-your-quizzes (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/users (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/badges/categories/all (ERRNO_ECONNREFUSED)
└─BROKEN─ https://quizzie.io/ (HTTP_undefined)
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
├─BROKEN─ https://quizzie.io/quizzes/50350/tout-ce-quil-faut-savoir-sur-la-creation-de-quiz (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/tags (HTTP_undefined)
├─BROKEN─ https://quizzie.io/quizzes/50300/markdown-ou-comment-ameliorer-la-presentation-de-vos-quiz (ERRNO_ECONNREFUSED)
├─BROKEN─ https://quizzie.io/users (HTTP_undefined)
├─BROKEN─ https://quizzie.io/badges/categories/all (HTTP_undefined)
├─BROKEN─ https://quizzie.io/ (HTTP_undefined)
```

**Comment by clacote** [comment]
Fixed by #293 and #294.
